modules/util/main/index.coffee

```diff
@@ -316,7 +316,7 @@ vendorPrefixes = ["webkit", "ms", "moz", "Moz", "o", "O"]
 hx.vendor = (obj, prop) ->
   if prop of obj then return obj[prop]
   for p in vendorPrefixes
-    if (prefixedProp = p + prop.charAt(0) + prop.slice(1)) of obj
+    if (prefixedProp = p + prop.charAt(0).toUpperCase() + prop.slice(1)) of obj
```

Comment:
It looks like it was always _supposed_ to do this (I can't think of another reason for it to get `charAt(0)` if not to capitalise) - Noticed as I was curious as to what hx.vendor actually did.

Added regression test(s)